flink-table-store-core/src/main/java/org/apache/flink/table/store/log/LogOptions.java

```diff
@@ -0,0 +1,192 @@
+            ConfigOptions.key("scan.timestamp-millis")
+                    .longType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "Optional timestamp used in case of \"from-timestamp\" scan mode");
+
+    public static final ConfigOption<Duration> RETENTION =
+            ConfigOptions.key("retention")
+                    .durationType()
+                    .noDefaultValue()
+                    .withDescription(
+                            "It means how long changes log will be kept. The default value is from the log system cluster.");
+
+    public static final ConfigOption<LogConsistency> CONSISTENCY =
+            ConfigOptions.key("consistency")
+                    .enumType(LogConsistency.class)
+                    .defaultValue(LogConsistency.TRANSACTIONAL)
+                    .withDescription(
+                            Description.builder()
+                                    .text("Specifies the log consistency mode for table.")
+                                    .linebreak()
+                                    .list(
+                                            formatEnumOption(LogConsistency.TRANSACTIONAL),
+                                            formatEnumOption(LogConsistency.EVENTUAL))
+                                    .build());
+
+    public static final ConfigOption<LogChangelogMode> CHANGELOG_MODE =
+            ConfigOptions.key("changelog-mode")
+                    .enumType(LogChangelogMode.class)
+                    .defaultValue(LogChangelogMode.AUTO)
+                    .withDescription(
+                            Description.builder()
+                                    .text("Specifies the log changelog mode for table.")
+                                    .linebreak()
+                                    .list(
+                                            formatEnumOption(LogChangelogMode.AUTO),
+                                            formatEnumOption(LogChangelogMode.ALL),
+                                            formatEnumOption(LogChangelogMode.UPSERT))
+                                    .build());
+
+    public static final ConfigOption<String> KEY_FORMAT =
+            ConfigOptions.key("key.format")
+                    .stringType()
+                    .defaultValue("json")
+                    .withDescription(
+                            "Specifies the key message format of log system with primary key.");
+
+    public static final ConfigOption<String> FORMAT =
+            ConfigOptions.key("format")
+                    .stringType()
+                    .defaultValue("debezium-json")
+                    .withDescription("Specifies the message format of log system.");
+
+    /** Specifies the startup mode for log consumer. */
+    public enum LogStartupMode implements DescribedEnum {
+        FULL(
+                "full",
+                "Performs a snapshot on the table upon first startup,"
```

Comment:
Nit: it seems like the descriptions are all started as third-person singular pronouns for `LATEST` and `FROM_TIMESTAMP`. We'd better keep consistency on this.